x-pack/legacy/plugins/apm/public/utils/formatters.ts

```diff
@@ -145,14 +148,14 @@ export function timeUnit(max: number) {
   }
 }
 
-export function asTime(
+export function asDuration(
   value: FormatterValue,
```

Comment:
A little off-topic but I got an idea and think it'll be a nice improvement. `FormatterValue` is currently defined as:
```ts
type FormatterValue = number | undefined | null;
```

We take this approach many places in the codebase where a type is defined as `T | null | undefined` (try searching for `null | undefined` and you'll find the other examples).

Currently it's not possible to see that `FormatterValue` expects a number. What about making a generic that makes this clear? 

```ts
type Maybe<T> = T | undefined | null;
```

Which can then be used like:
```suggestion
   value: Maybe<number>,
```

If this is too much for this PR that's fair.